### PR TITLE
Update UI and email copy

### LINKED ISSUE #61: Review copy in emails and UI notifications
New pledge form

* [x] Pledge submitted ([#](https://github.com/WordPress/five-for-the-future/blob/dbdcf26bd9dbc8f557fb5036f07675606fb43def/plugins/wporg-5ftf/views/form-pledge-new.php#L33-L49))
* [x] Pledge email address confirmed ([#](https://github.com/WordPress/five-for-the-future/blob/dbdcf26bd9dbc8f557fb5036f07675606fb43def/plugins/wporg-5ftf/views/form-pledge-confirm-email.php#L16-L18))

My pledges page

* [x] Notices ([#](https://github.com/WordPress/five-for-the-future/blob/5b5a1d50ffb564bf071a1db0941005bae60895d8/plugins/wporg-5ftf/includes/contributor.php#L249-L266))

Emails

* [x] Pledge email confirmation ([#](https://github.com/WordPress/five-for-the-future/blob/dbdcf26bd9dbc8f557fb5036f07675606fb43def/plugins/wporg-5ftf/includes/pledge.php#L157-L161))
* [x] Contributor confirmation ([#](https://github.com/WordPress/five-for-the-future/blob/dbdcf26bd9dbc8f557fb5036f07675606fb43def/plugins/wporg-5ftf/includes/pledge-form.php#L167-L183))

### PR DESCRIPTION
**Pull request by coreymckrill**
Fixes #61

### CODE REVIEW

File: plugins/wporg-5ftf/includes/pledge-form.php [hunk 1]
```diff
@@ -95,11 +95,11 @@ function process_form_new() {
 	Contributor\add_pledge_contributors( $new_pledge_id, $contributors );
 
 	// Attach logo to the pledge.
-	wp_update_post( array(
+	/*wp_update_post( array(
 		'ID'          => $logo_attachment_id,
 		'post_parent' => $new_pledge_id,
 	) );
-	set_post_thumbnail( $new_pledge_id, $logo_attachment_id );
+	set_post_thumbnail( $new_pledge_id, $logo_attachment_id );*/
```
Comment (iandunn) [inline plugins/wporg-5ftf/includes/pledge-form.php:102]: @coreymckrill I'm guessing we'll need to reactivate this for other pledges to be submitted today, is that right?

Comment (coreymckrill) [reply plugins/wporg-5ftf/includes/pledge-form.php:102]: Ah crap.

Comment (coreymckrill) [reply plugins/wporg-5ftf/includes/pledge-form.php:102]: Good catch

Comment (coreymckrill) [reply plugins/wporg-5ftf/includes/pledge-form.php:102]: Fixed in 454962522db7b8355c95d048496f1eeeab2c478f

**Pull request merged**